app/webpack/projects/show/components/view_more_footer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Grid, Row, Col } from "react-bootstrap";
+
+/**
+ * Renders a button-styled link with the text "View More"
+ * if the prop showViewMoreLink is true.
+ *
+ * Otherwise, renders nothing.
+ *
+ * Spacing around the link is appropriate for use as a footer.
+ *
+ */
+
+const ViewMoreFooter = ( { showViewMoreLink, viewMoreUrl } ) => {
+    return (showViewMoreLink
+        ?
+        <div className="ViewMoreFooter">
+            <Grid>
+                <Row>
+                    <Col xs={12}>
+                        <a href={viewMoreUrl} className={"btn btn-primary btn-inat btn-green pull-left"}>{ I18n.t( "view_more" ) }</a>
```

Comment:
Can you remove the `pull-left` here and add `className="text-center"` to the <Col> so the button is centered?

Reply:
Done! Thx for the review.

Reply:
Actually, I'm not sure that's sufficient from playing around with CSS on other buttons on the site. I'll figure it out & get back to you later this week.

Reply:
Oops, forgot some backticks. Yeah, `className="text-center"` should be on a surrounding element.

Reply:
Fixed for real & tested.